Pass the -q flag to `login(1)` on macOS, when `~/.hushlogin` file is present (#8490)
alacritty/alacritty: alacritty_terminal/CHANGELOG.md

## 0.25.1-dev

### Changed

> -  Shell initialization on macOS to manually check the `~/.hushlogin` file

Reviewer: ```suggestion
- Pass `-q` to `login` on macOS if `~/.hushlogin` is present
```
